### ISSUE TITLE
Icon only button examples should include aria-labels

### ISSUE THREAD

**Issue by sowasred2012** [canonical-web-and-design/vanilla-framework#4033]
We have a few examples of buttons that only have icons, and lack any labelling text, such as `aria-label` attributes.

[ARIA live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) may also be a factor for the two examples below - presumably in the implementation the button contents will change depending on whatever process they represent processing/completing.

![Screenshot from 2021-10-12 16-30-07.png](https://images.zenhubusercontent.com/5cd93aaf2406d87914830598/22ece851-b86e-4c30-b8d5-89fc9c438df6)

**Comment by sowasred2012** [comment]
@anthonydillon yeah, we were just talking about this, and is why I've just created a bunch of issues, including https://github.com/canonical-web-and-design/vanilla-framework/issues/4032 which relates to your example in particular - basically, we realised we could achieve what `%vf-hide-text` does using `aria-label` attributes (on the button, in your example), so CSS doesn't have to get involved when making these things accessible.

I don't think we necessarily want to remove `%vf-hide-text` in 3.0 - because maybe it'd be a big pain when upgrading - so we landed on just documenting it this way, and keeping support for the way we've had it up to now.
